Example/CustomActionControllers/Spotify.swift

```diff
@@ -137,35 +137,69 @@ open class SpotifyHeaderView: UICollectionReusableView {
     }
 }
 
-open class SpotifyActionController: ActionController<SpotifyCell, ActionData, SpotifyHeaderView, SpotifyHeaderData, UICollectionReusableView, Void> {
+open class SpotifyActionController: ActionController<SpotifyCell, ActionData, SpotifyHeaderView, SpotifyHeaderData, UICollectionReusableView, Void, UICollectionReusableView> {
     
     fileprivate lazy var blurView: UIVisualEffectView = {
         let blurView = UIVisualEffectView(effect: UIBlurEffect(style: .dark))
         blurView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
         return blurView
     }()
     
+    var cancelView: UIView?
+    
     open override func viewDidLoad() {
         super.viewDidLoad()
+        
         backgroundView.addSubview(blurView)
+        cancelView = UIView(frame: CGRect(x: 0, y: view.bounds.size.height - 60, width: UIScreen.main.bounds.width, height: 60))
+        
+        cancelView = {
+            let cancel = cancelView!
+            cancel.autoresizingMask = [.flexibleWidth, .flexibleTopMargin]
+            cancel.backgroundColor = .black
+            
+            cancel.layer.shadowColor = UIColor.black.cgColor
+            cancel.layer.shadowOffset = CGSize( width: 0, height: -4)
+            cancel.layer.shadowRadius = 2
+            cancel.layer.shadowOpacity = 0.8
+
+            let cancelButton: UIButton = {
+                let cancelButton = UIButton(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: 60))
+                cancelButton.addTarget(self, action: #selector(SpotifyActionController.cancelButtonDidTouch(_:)), for: .touchUpInside)
+                cancelButton.setTitle("Cancel", for: UIControlState())
+                cancelButton.translatesAutoresizingMaskIntoConstraints = false
+                return cancelButton
+            }()
+            cancel.addSubview(cancelButton)
+            cancel.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|[button]|", options: NSLayoutFormatOptions(), metrics: nil, views: ["button": cancelButton]))
+            cancel.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|[button]|", options: NSLayoutFormatOptions(), metrics: nil, views: ["button": cancelButton]))
+            return cancel
+        }()
+        
+        view.addSubview(cancelView!)
+    
+        calculateContentInset()
+    }
+    
+    open override func setUpContentInsetForHeight(_ height: CGFloat) {
```

Comment:
If setUpContentInsetForHeight is not override the last cell of the collectionView is covered with the cancelButton. So we need to set the bottomInset of the collectionView with the height of the cancel button.